calc/vxlan_resolver.go

```diff
@@ -423,25 +490,80 @@ func (c *VXLANResolver) routeReady(r vxlanRoute) bool {
 		logCxt.Debug("Don't yet know the VTEP for this route")
 		return false
 	}
+
+	if _, ok := c.nodeNameToNode[c.hostname]; !ok {
```

Comment:
Hm, I think this won't work in the scenario where we're using an old-version of Typha with a new version of Felix, since the old version of Typha will never send through the `v3.Node` object and so this check will never pass.

Reply:
ah right, I meant to check what notifications we're accepting before doing this. This is only meant to happen if we have the resource updates flag set